doc/services/portability/posix/option_groups/index.rst

```diff
@@ -403,7 +403,7 @@ _POSIX_READER_WRITER_LOCKS
     pthread_rwlockattr_destroy(),yes
     pthread_rwlockattr_getpshared(),
```

Comment:
This should probably be implemented in the same PR. You can fix two issues for the price of one 😀